lib/internal/streams/operators.js

```diff
@@ -147,6 +147,20 @@ async function * map(fn, options) {
   }
 }
 
+async function * filter(fn, options) {
+  if (typeof fn !== 'function') {
+    throw (new ERR_INVALID_ARG_TYPE(
+      'fn', ['Function', 'AsyncFunction'], this));
```

Comment:
```suggestion
    throw new ERR_INVALID_ARG_TYPE(
      'fn', ['Function', 'AsyncFunction'], this);
```
nit: unnecessary wrapping, most of the methods that throw a new instance of an error does not wrap them.

Reply:
Yeah this is just leftover good catch